AspNet.Security.OpenIdConnect.Samples.sln

```diff
@@ -13,6 +13,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SignalR", "SignalR", "{61F3
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Mvc", "Mvc", "{139182EF-E215-4E9C-91AA-312F29B33A76}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{57AA233F-DD3B-4971-9C24-D024E07656EE}"
```

Comment:
I'm not a huge fan of these default items, but if you think they are important, feel free to keep them.